src/pubtools/_marketplacesvm/cloud_providers/aws.py

```diff
@@ -50,7 +57,10 @@ def get_2_digits(version: str) -> str:
     underscore_parts = []
 
     if release.version is not None:
-        underscore_parts.append(get_2_digits(release.version))
+        if ami_version_template:
+            underscore_parts.append(format_version(release.version, ami_version_template))
+        else:
+            underscore_parts.append(get_2_digits(release.version))
```

Comment:
`get_2_digits` should be equivalent of 
```
format_version(release.version, "{major}.{minor}")
```
maybe we could change the default value of the parameter `ami_version_template` to such value :-)